.github/workflows/ci.yml

```diff
@@ -26,16 +26,20 @@ jobs:
       
       # Sets up MATLAB on the GitHub Actions runner
       - name: Setup MATLAB
-        uses: matlab-actions/setup-matlab@v1
+        uses: matlab-actions/setup-matlab@v2
 
       # Runs a set of commands using the runners shell
       - name: Run all tests
-        uses: matlab-actions/run-tests@v1
+        uses: matlab-actions/run-tests@v2
         with:
           source-folder: code
 
+      # You can use "run-build" to invoke the MATLAB build tool and run build tasks
+      # - name: Run tests with default build tool task
+      #   uses: matlab-actions/run-build@v2
+
```

Comment:
On line 37, there is no period. On line 41, there is one. Suggest removing it for consistency.